README.md

```diff
@@ -181,9 +181,23 @@ python3 labours.py [--text-size] [--relative]
 1. Currently, go-git's file system storage backend is considerably slower than the in-memory one,
 so you should clone repos instead of reading them from disk whenever possible. Please note that the
 in-memory storage may require much RAM, for example, the Linux kernel takes over 200GB in 2017.
-2. Parsing YAML in Python is slow when the number of internal objects is big. `hercules`' output
+1. Parsing YAML in Python is slow when the number of internal objects is big. `hercules`' output
```

Comment:
Yes, it is able to render the ordered lists this way.

You can check out how it looks like https://github.com/bzz/hercules/blob/1a43f1bfd0f6c369e298e95522843b538b20d875/README.md#caveats